### PR TITLE
Disable flaky tests. (uplift to 1.19.x)

### PR DESCRIPTION
**Pull request by zenparsing**
Uplift of #7312

Approved, please ensure that before merging: 
- [x] You have checked CI and the builds, lint, and tests all pass or are not related to your PR. 
- [ ] You have tested your change on Nightly. 
- [x] The PR milestones match the branch they are landing to. 

After you merge: 
- [ ] The associated issue milestone is set to the smallest version that the changes is landed on.

### CODE REVIEW

**Review by emerick — APPROVED** [review]
LGTM

**Review by kjozwiak — APPROVED** [review]
Uplift into `1.19.x` approved after deliberating with @brave/uplift-approvers.

**Pull request merged**
